dig.go

```diff
@@ -258,6 +262,13 @@ func setRand(r *rand.Rand) Option {
 	})
 }
 
+// Dry sets container mode to dry used for testing graphs without invocation.
+func Dry(dry bool) Option {
```

Comment:
1) Are we attached to the name `Dry` for this option? I feel like `DryRun` or even `ValidateGraphOnly` might be a more descriptive name. 
2) Nice; I like accepting the bool argument here. I could see an argument for having the API be just whether or not you provide the `Dry` option, but I like the ability to pass in, say, environment variables as arguments.

Reply:
Dry is a pretty common option https://en.wikipedia.org/wiki/Dry_run_(testing) and is present for most of git commands, for example, with my understanding that it signals that nothing is going to run and we are testing. And yeah, usually it's a boolean.

Reply:
Right; wikipedia has it as "dry run" like I suggested. it's not a huge deal, but I think `DryRun(true)` is clearer than `Dry(true)` for the api. 

And for the second comment, I meant more that in my CLI experience, the API can either be an explicit boolean — ` --dry-run=true` — or implicit just by adding the flag ` --dry-run`. I guess I'm saying that here, the `dig.Option` could be `Dry()` without accepting the bool argument, or `Dry(true/false)` .  I was just commenting that I agree with the choice to explicitly pass the value.